packages/experimental/TabList/src/TabList.types.ts

```diff
@@ -35,7 +34,12 @@ export interface TabListContextData {
   focusZoneRef?: React.RefObject<any> | null;
 }
 
-export interface TabListTokens extends IForegroundColorTokens, FontTokens, IBackgroundColorTokens {}
+export interface TabListTokens {
+  /**
+   * Flex direction of Tab items. Controlled by TabList `vertical` prop.
+   */
+  flexDirection?: ViewStyle['flexDirection'];
```

Comment:
flexDirection shouldn't be needed as a token if we have vertical as a prop

Reply:
Vertical is the prop that would drive the styling but flexDirection needs to be a specified token for the appropriate styling to be applied when vertical is set.

The flow of props to styles is generally the following:
Prop applies a layer of style tokens so that the output of tokens has the appropriate values given the current set of props.
Style tokens are applied at the useStyling stage and converted into props for each slot.
Slot props are passed to slots at inner render.

Reply:
@lawrencewin I would say that it doesn't make sense for vertical to be a token, since "vertical" is not a style. Vertical should be a prop which applies style tokens.